docs/src/main/asciidoc/rest-data-panache.adoc

```diff
@@ -431,6 +431,8 @@ In this example, we have added a named query to list all the persons that contai
 
 Next, we can set a query param `namedQuery` when listing the entities using the generated resource with the name of the named query that we want to use, for example, calling `http://localhost:8080/people?namedQuery=Person.containsInName&name=ter` would return all the persons which name contains the text "ter".
 
+IMPORTANT: If you have pagination enabled, using JOIN FETCH or others might prevent the query (`namedQuery`) from being reused for the count query. To resolve this you can add an additional query by adding a comma in front of it like this `http://localhost:8080/people?namedQuery=Person.containsInName,Person.countContainsInName&name=ter` which will be used for the pagination count.
```

Comment:
I don't see any tests that exercise/cover this. Can you add one?
Also, I don't fully understand the limitation you describe here, but perhaps I could understand it better with an example in the test. 
In any case, I don't feel that using an extra comma is super intuitive, so perhaps a new namedQueryForCount query param would be a better solution.